lib/stormpath-sdk/resource/base.rb

```diff
@@ -57,6 +57,10 @@ def non_printable_properties
       @non_printable_properties ||= []
       Array.new @non_printable_properties
     end
+
+    def has_status(name: :status, status_list: :default)
```

Comment:
@ecrisostomo I'm thinking of actually moving this - if somebody sets the status to something that's not defined, the server will throw an error. What do you think about that?